### DISCUSSION TITLE
libCbc build in VS2008 (v9)

### DISCUSSION THREAD

**Discussion by b3nbr0wn** [coin-or/Cbc#651] [Q&A]
Hello there,

I am attempting to compile CBC in Visual Studio 2008 (v9) but not having success.  I realize this is not an ideal build environment, but this is a requirement in my particular project.

I am curious if the MSVisualStudio/v9 projects are still maintained and viable?  I have had to update project include paths to include some missing subfloders (ie CglCommon), etc.

Should I continue updating projects and source until I get a successful build?  Or am I missing something more fundamental?

Any help would be appreciated!

**Comment by jhmgoossens** [comment]
To the best of my knowledge, only the latest VS 2022 projects are maintained. 

Manually fixing the old VS 2008 projects could work. Use the VS 2022 project files for hints for paths etc. Create tickets if you run into issues. 

Does this help?

**Reply by b3nbr0wn** [reply]
Yes, this is helpful, thank you.

**Comment by b3nbr0wn** [comment]
For anyone else attempting to compile CBC in VS2008, I was able to do so successfully by
- pull the most recent **stable** branches of the required coin repositories
- update Cbc.sln to reference new paths to .vcproj files
- update .vcproj file's AdditionalIncludeDirectories with new paths to src and header folders
